import-export-cli/integration/api_test.go

```diff
@@ -397,6 +397,58 @@ func TestExportImportApiCrossTenantUserWithoutPreserveProvider(t *testing.T) {
 	testutils.ValidateAPIImport(t, args)
 }
 
+// Import an API with the default version. Change the version and import the same API again.
+// For tenant user.
+func TestTenantUserApiVersioning(t *testing.T) {
+
+	tenantDevopsUsername := devops.UserName + "@" + TENANT1
+	tenantDevopsPassword := devops.Password
+
+	tenantApiCreator := creator.UserName + "@" + TENANT1
+	tenantApiCreatorPassword := creator.Password
+
+	dev := apimClients[0]
+	prod := apimClients[1]
+
+	api := testutils.AddAPI(t, dev, tenantApiCreator, tenantApiCreatorPassword)
+
+	args := &testutils.ApiImportExportTestArgs{
+		ApiProvider:      testutils.Credentials{Username: tenantApiCreator, Password: tenantApiCreatorPassword},
+		CtlUser:          testutils.Credentials{Username: tenantDevopsUsername, Password: tenantDevopsPassword},
+		Api:              api,
+		OverrideProvider: true,
+		SrcAPIM:          dev,
+		DestAPIM:         prod,
+	}
+
+	testutils.ValidateAPIExport(t, args)
+
+	args.ApiProvider = testutils.Credentials{Username: tenantDevopsUsername, Password: tenantDevopsPassword}
```

Comment:
If we haven't override the provider as mentioned above, we don't need this assignment as well

Reply:
Fixed